test/cypress/integration/request-to-go-live/organisation_name_spec.js

```diff
@@ -3,6 +3,7 @@ describe('Request to go live: organisation name', () => {
 
   describe('NO PERMISSIONS', () => {
     beforeEach(() => {
+      cy.clearCookies()
```

Comment:
I don't think clearing the cookies is necessary as they will automatically be cleared between each test